CoreTweet.Shared/Internal/Converters.cs

```diff
@@ -248,24 +248,24 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             switch (reader.TokenType)
             {
                 case JsonToken.String:
-                    return DateTimeOffset.ParseExact(reader.Value as string, "yyyyMMddHHmm",
-                                                     DateTimeFormatInfo.InvariantInfo,
-                                                     DateTimeStyles.AllowWhiteSpaces);
+                    return DateTime.ParseExact(reader.Value as string, "yyyyMMddHHmm",
+                                               DateTimeFormatInfo.InvariantInfo,
+                                               DateTimeStyles.AllowWhiteSpaces);
                 case JsonToken.Date:
-                    if (reader.Value is DateTimeOffset)
-                        return (DateTimeOffset)reader.Value;
+                    if (reader.Value is DateTime)
+                        return (DateTime)reader.Value;
                     else
-                        return new DateTimeOffset(((DateTime)reader.Value).ToUniversalTime(), TimeSpan.Zero);
+                        return ((DateTime)reader.Value).ToUniversalTime();
                 case JsonToken.Integer:
-                    return DateTimeOffset.ParseExact(reader.Value.ToString(), "yyyyMMddHHmm",
-                                                     DateTimeFormatInfo.InvariantInfo,
-                                                     DateTimeStyles.AllowWhiteSpaces);
+                    return DateTime.ParseExact(reader.Value.ToString(), "yyyyMMddHHmm",
+                                               DateTimeFormatInfo.InvariantInfo,
+                                               DateTimeStyles.AllowWhiteSpaces);
 
                 case JsonToken.Null:
-                    return DateTimeOffset.Now;
+                    return DateTime.Now;
```

Comment:
This breaks down the content of the original response.

Reply:
I cannot judge what it should be because I cannot use Premium APIs. So please review the response from Twitter. If the fields (`fromDate`, `toDate` and `timePeriod`)  can be null or can lack, the field should be `DateTime?`, otherwise keep them `DateTime`. In the converter, you can return `null` regardless of whether the `objectType` is nullable or not.